MiniApp/Classes/core/JavascriptBridge/MiniAppScriptMessageHandler.swift

```diff
@@ -491,6 +493,21 @@ internal class MiniAppScriptMessageHandler: NSObject, WKScriptMessageHandler {
         }
     }
 
+    func getHostEnvironmentInfo(with callbackId: String) {
+        hostAppMessageDelegate?.getHostEnvironmentInfo(completionHandler: { (result) in
+            switch result {
+            case .success(let response):
+                guard let encodedResult = ResponseEncoder.encode(data: response) else {
+                    self.executeJavaScriptCallback(responseStatus: .onError, messageId: callbackId, response: getMiniAppErrorMessage(MiniAppJavaScriptError.internalError))
```

Comment:
@laubengaier We need to use `prepareMAJavascriptError` instead of `getMiniAppErrorMessage` to send response in JSON format.

Reply:
@rleojoseph thanks, I'll replaced it and made the method `handleMASDKErrorWithJson`that uses the prepare method. hope the method name is ok & descriptive